source/boards/data-and-archives.rst

```diff
@@ -0,0 +1,106 @@
+Import your data
+================
+
+You can import data from other tools to use with Boards.
+
+Import from Asana
+-----------------
+
+This node app converts an Asana JSON archive into a ``.focalboard`` archive.
+
+1. Log into your Asana account.
+2. From the Asana Board Menu, select **Export/Print > JSON** to create an archive file (which you'll use in Boards).
+3. Save the file locally, e.g. to ``asana.json``.
+4. In your local copy of Boards, open a terminal window.
```

Comment:
Is this correct? If not, where and how would the user access terminal to enter this command?

Reply:
Sort of...it really isn't "In you local copy of Boards". (You can probably drop that portion.) But it's just opening a window to run a command prompt. Would be different depending on OS. Mac you can run `terminal`. I think windows is `cmd`. There are also 3rd party packages for this.